app/models/shelter.rb

```diff
@@ -20,6 +20,13 @@ def self.order_by_desc_alphabetically
     find_by_sql("SELECT shelters.* FROM shelters ORDER BY name desc")
   end
 
+  def self.shelters_by_status
+    joins(pets:[:applicants]).where({ "applicants.status" => 'Pending'}).distinct
+    #     select shelters.name, pets.id, pets.name, applicants.first_name, applicants.id from shelters join pets on she
+    # lters.id = pets.shelter_id join applicant_pets on pets.id = applicant_pets.pet_id join applicants on applicants.id = applicant_pets.applica
+    # nt_id where applicants.status like 'Pending';
```

Comment:
This was really cool to figure out first in SQL and then figure out how that translates to AR with the multiple join tables. I'm really impressed with how we worked through this part.